### DISCUSSION TITLE
scriptable rendering

### DISCUSSION THREAD

**Discussion by aol06-skyuk** [requarks/wiki#6083] [Help / Questions] [answered]
Hi,

After a db restore, pages are still showing the old content just as the restore never took place.

Bouncing the docker container does not help, the pre-restore content is still showing.

"SELECT content FROM pages;" in postgres show the correct (restored) content but only after manually rendering the page, the content is also displayed correctly in the browser.

Manually rendering all pages is time consuming so I was wondering if there is a scriptable way to force rendering of all pages from the command line but I wasn't able to find relevant info in the discussions hence the question:
is there a scriptable way to force rendering of all pages?

Cheers,
Andrea

**Comment by NGPixel** [comment, marked as answer]
Admin > Utilities > Content > Rerender all pages?

**Reply by aol06-skyuk** [reply]
Thank you Nicolas, that certainly helps.
So I should get away with curl calls and jq
